Packs/TrendMicroDDA/Integrations/TrendMicroDDA/TrendMicroDDA_test.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+import demistomock as demisto
+
+
+@pytest.mark.parametrize('sample_url', ["https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DgrBFMP3HDZA"])
+def test_simple_upload_sample_url(mocker, requests_mock, sample_url):
```

Comment:
please just add explanation to what is being tested here.
also, for 1 test, there is no need to parametrize.

Reply:
Done